packages/visx-axis/src/types.ts

```diff
@@ -11,6 +12,8 @@ export type AxisScale<Output extends AxisScaleOutput = AxisScaleOutput> =
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   D3Scale<Output, any, any>;
 
+type LineProps = Omit<React.ComponentProps<typeof Line>, 'to' | 'from'>;
```

Comment:
oof, looks like this somehow results in a very complex type such that TS throws

![image](https://user-images.githubusercontent.com/4496521/118333702-03c71a80-b4c1-11eb-9d92-41fe59024533.png)

I think if we simplify this to just `type LineProps = Omit<React.SVGProps<SVGLineElement>, 'to' | 'from'>;` (TS seems to throw if we don't omit `to/from` because the types are different for the native `SVGLineElement`) and remove the `import Line from '@visx/shape/lib/shapes/Line';` line above we should be good. Sorry for the back and forth on this!

Reply:
I have fixed this :)